cmd/canary/canary-config/save.go

```diff
@@ -78,7 +78,7 @@ func saveCanaryConfig(cmd *cobra.Command, options SaveOptions) error {
 	var saveErr error
 	if resp.StatusCode == http.StatusOK {
 		_, saveResp, saveErr = gateClient.V2CanaryConfigControllerApi.UpdateCanaryConfigUsingPUT(
-			gateClient.Context, templateId, templateJson, map[string]interface{}{})
+			gateClient.Context, templateJson, templateId, map[string]interface{}{})
```

Comment:
This was the one non automated change. The order of the arguments to the api helper function had to be swapped.

Reply:
Just curious, but why was that? Did the order change in the gate API or something?

Reply:
Created an issue to track this, https://github.com/spinnaker/spinnaker/issues/4931